.mailmap

```diff
@@ -209,6 +209,11 @@ Abhinav Anand <abhinav.anand2807@gmail.com>
 Abhinav Chanda <abhinavchanda01@gmail.com>
 Abhishek <uchiha@pop-os.localdomain>
 Abhishek Garg <abhishekgarg119@gmail.com>
+Abhishek Patidar <1e9abhi1e10@gmail.com> 1e9abhi1e10 <1e9abhi1e10@gmaiil.com>
+Abhishek Patidar <1e9abhi1e10@gmail.com> <1e9abhi1e10@gmail.com>
+Abhishek Patidar <1e9abhi1e10@gmail.com> ABHISHEK PATIDAR <95904102+1e9abhi1e10@users.noreply.github.com>
+Abhishek Patidar <1e9abhi1e10@gmail.com> Abhishek Patidar <2311abhiptdr@gmail.com>
+Abhishek Patidar <1e9abhi1e10@gmail.com> abhi1e9-1e10 <1e9abhi1e10@gmail.com>
```

Comment:
Currently this part has merge conflicts but really these lines do not need to be added to .mailmap. Your correct name and email address is already in the .mailmap file from another PR. If you just start a new PR and commit with the correct name and email address then these lines won't be needed any more.